### DISCUSSION TITLE
Debugging : Logging + Viewing print statements

### DISCUSSION THREAD

**Discussion by manu-web** [ggml-org/llama.cpp#8376] [Q&A]
How does --log-test option work? I see no difference in output. And how can I see the output of print statements? I gave the --verbose option but could not see

**Comment by dspasyuk** [comment]
Try using  --logdir ./  --log-enable it will create log files in ./ folder you should have everything you need in main.log file

**Reply by manu-web** [reply]
I actually wanted output to my print statements. It seems that they are not coming to stdout and have been redirected.

**Reply by dspasyuk** [reply]
Yes, look at common/log.h for more details, llama.cpp prints a lot of logs. I am not sure you want that. What logs are you looking for?

**Reply by manu-web** [reply]
I was not able to see the prints that I have added in fattn-common.cuh but was able to use GGML_LOG_INFO ... in ggml_cuda.cu. Using it for some debugging. How do we see timestamps printed in the fattn-common.cuh?
